src/components/collection/Selector/List/index.tsx

```diff
@@ -81,10 +83,18 @@ function CollectionSelectorList({
     const [notificationMessage, setNotificationMessage] = useState('');
     const [showNotification, setShowNotification] = useState(false);
 
+    // Form State Store
+    const formStatus = useFormStateStore_status();
+
+    // Resource Config Store
     const collections = useResourceConfig_collections();
     const currentCollection = useResourceConfig_currentCollection();
 
-    const selectionEnabled = currentCollection && setCurrentCollection;
+    const selectionEnabled =
+        currentCollection &&
+        setCurrentCollection &&
+        formStatus !== FormStatus.UPDATING;
```

Comment:
We can restrict this condition further by applying the form status condition only when the entity type is a materialization. Presently, the binding selector will also be locked while the capture auto discovery settings are updated.

Reply:
Reducing scope to just materializations kinda makes sense in terms of risk but then I think it makes the app kinda weird feeling. I think a consistently "worse" experience is better than a inconsistent experience.